assets/js/components/BatchEdit/About/UncontrolledMetadata.test.jsx

```diff
@@ -6,20 +6,30 @@ import {
 } from "../../../services/testing-helpers";
 import BatchEditAboutUncontrolledMetadata from "./UncontrolledMetadata";
 import { UNCONTROLLED_METADATA } from "../../../services/metadata";
+import { allCodeListMocks } from "@js/components/Work/controlledVocabulary.gql.mock";
+import { CodeListProvider } from "@js/context/code-list-context";
 
 describe("BatchEditAboutUncontrolledMetadata component", () => {
   beforeEach(() => {
     const Wrapped = withReactHookForm(BatchEditAboutUncontrolledMetadata);
-    return renderWithRouterApollo(<Wrapped />);
+    return renderWithRouterApollo(
+      <CodeListProvider>
+        <Wrapped />
+      </CodeListProvider>,
+      {
+        mocks: allCodeListMocks,
+      }
+    );
   });
 
-  it("renders the component", () => {
-    expect(screen.getByTestId("uncontrolled-metadata"));
+  it("renders the component", async () => {
+    expect(await screen.getByTestId("uncontrolled-metadata"));
```

Comment:
`@testing-library/react` async selectors are kinda specific.  To get this to properly evaluate, you'll need to swap out `getByTestId` with `findByTestId`.  Anytime you front load the expression with `await`, you'll need to use the `findBy...`.

Reply:
Didn't know that, thanks for the heads up! Will fix.

Reply:
@adamjarling pushed the `findByTestId` version, thanks for the feedback 😄